GameDefinition/PLAYING.md

```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
 of exploration.
 
 
+Managing inventory and crafting
+-------------------------------
+
+All items belonging to your team are stored either in the character's personal 
+equipment outfit or in the shared inventory stash that is accessible by every 
+team member. 
+Items that are in the personal equipment outfit can be accessed by or 
+are beneficial for this particular character only. Stashed items can be 
+accessed by all the team members except those that are not calm enough 
+or injured somehow. But no one has a direct advantage from having the items 
+in the shared inventory stash.
+The stashed items are exactly stashed, they are not simply laying on
+the ground (although the stash has a particular location on the map). 
+To operate the stashed items, you should use commands that refer to 
+the party shared inventory, not to items on the ground.
+
+Obviously, if the item is beneficial (a piece of armor, weapon, or a bonus
+item), it is reasonable to put it into a particular character's outfit.
+However, you should remember that the outfit is limited to any 10 (ten)
+items only. On the contrary, if the item is not required directly (ranged
+weapon, medkits, ampoules with drugs, food, or else), it may be stored in
+the shared inventory stash.
+
+The shared inventory stash is of unlimited size. It has a physical location on
+the map and may be looted. So, it is wise to keep it safe or change 
+its location depending on the current situation. It would be a pity to lose 
+valuables or powerful projectiles by not taking care of the shared inventory. 
+
+To relocate the shared inventory stash, first, you need to remove ALL 
+of the items from it anywhere you want to make it empty. Then, any stashed
+item (or a pile of items) will automatically become the new stash. The process 
+takes a couple of turns, so you should plan shared inventory stash relocation
+in advance.
+You should keep your eye on the item management stat in the character menu
+`#`. If this stat is too low (which may be caused by an injury or low 
+calmness level), it will be difficult for the character to work with items in 
+the character's personal outfit or in the shared inventory stash.
```

Comment:
to be precise, the character can still melee with the items, so perhaps: "add or remove items to the character's personal outfit". This also covers moving items from stash to outfit, so the "or in the shared inventory stash" can be skipped.

Reply:
Done